client/stylesheets/_toCat.scss

```diff
@@ -5067,6 +5114,84 @@ On click the images will be displayed at normal size to complete the effect
 
 /* End */
 
+/* Header Location Radius */
+
+.headerLocationRadius {
+	float: left;
+	margin-left: 16px !important;
+	margin-top: 8px !important;
+	vertical-align: middle !important;
+	font-size: 0.9rem;
+}
+
+.headerRadius {
+	width: 70px;
+	margin-left: 10px;
+	padding: 8px 2px;
+	color: #fff !important;
+	background-color: #f73c37;
+	text-align: center;	
+}
+
+.headerRadius input {
+	width: 16px;
+	display: inline-block;
+	-webkit-appearance: none !important;
+    -moz-appearance: none !important;
+    appearance: none !important;
+    text-align: center;
+    outline: none !important;
+    background-color: #f73c37;
+	color: #fff;
+	border: 1px solid #f73c37;
+	margin-left: -4px;
+	font-family: 'Droid Sans', sans-serif;
+}
+
+.headerRadius span {
+	display: inline-block;
+	margin-left: 0;
+	padding-right: 4px;
+}
+
+/*
+.headerRadius {
+	margin-left: 10px;
+	max-width: 30px;
+	background-color: #f73c37;
+	color: #fff;
+	border: 1px solid #f8504b;
+	padding: 5px;
+}
+
+.headerRadius input {
+	-webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    text-align: center;
+    padding: 0;
+    outline: none;
+    background-color: #f73c37;
+	color: #fff;
+} */
+
+.headerRadius input:focus {
+	outline: none !important;
+	-webkit-appearance: none !important;
+    -moz-appearance: none !important;
+    appearance: none !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces